src/utils/utils.js

```diff
@@ -0,0 +1,33 @@
+export function fetchData(url) {
+	return fetch(url)
+		.then((result) => result.json())
+		.catch((error) => error);
+}
+
+export function postData(url, taskname, description) {
+	return fetch(url, {
+		method: "POST",
+		headers: { "content-type": "application/JSON" },
+		body: JSON.stringify({
+			task_name: taskname,
+			description: description,
+			completed: false,
+		}),
+	})
+		.then((result) => result.json())
+		.catch((error) => error);
+}
+
+export function paramsUrl(url, taskname, description) {
+	const newUrl = `${url}?task_name=${taskname}&description=${description}&completed=false`;
+	return newUrl;
+}
+
+export function sortDateTime(date) {
```

Comment:
This is a language thing and I might not get it, but why `sort...`? I always use `sort` in names for sorting, that is putting things in a particular order, not modifying data.

Reply:
Yes I used the term sort because I'm sorting the order of the date and time string into separate items.